app/services/dqt/trn_request_params.rb

```diff
@@ -0,0 +1,123 @@
+        address: nil, # we don't collect address
+        genderCode: "NotAvailable",
+        initialTeacherTraining: initial_teacher_training_params,
+        qualification: qualification_params,
+      }
+    end
+
+    private
+
+    attr_reader :application_form, :teacher
+
+    #   providerUkprn: (string)
+    #   programmeStartDate: (string, required)
+    #   programmeEndDate: (string, required)
+    #   programmeType: (string, required) https://github.com/DFE-Digital/qualified-teachers-api/blob/main/docs/api-specs/v2.json#L943-L968
+    #   subject1: (string)
+    #   subject2: (string)
+    #   subject3: (string)
+    #   ageRangeFrom: (integer)
+    #   ageRangeTo: (integer)
+    #   ittQualificationType: https://github.com/DFE-Digital/qualified-teachers-api/blob/main/docs/api-specs/v2.json#L991-L1060
+    #   ittQualificationAim: https://github.com/DFE-Digital/qualified-teachers-api/blob/main/docs/api-specs/v2.json#L983-L987
+    #   ittQualificationType: https://github.com/DFE-Digital/qualified-teachers-api/blob/main/docs/api-specs/v2.json#L1019-L1059
+    def initial_teacher_training_params
+      {
+        providerUkprn: nil,
+        programmeStartDate: teaching_qualification.start_date,
+        programmeEndDate: teaching_qualification.end_date,
+        programmeType: "OverseasTrainedTeacherProgramme",
+        subject1:  application_form.subjects[0],
+        subject2:  application_form.subjects[1],
+        subject3:  application_form.subjects[2],
+        ageRangeFrom: application_form.age_range_min,
+        ageRangeTo: application_form.age_range_max,
+      }
+    end
+
+    #https://github.com/DFE-Digital/qualified-teachers-api/blob/main/docs/api-specs/v2.json#L803-L837
+    # qualification: (object)
+    #   providerUkprn: (string)
+    #   countryCode: (string)
+    #   subject: (string)
+    #   subject2: (string)
+    #   subject3: (string)
+    #   class: https://github.com/DFE-Digital/qualified-teachers-api/blob/main/docs/api-specs/v2.json#L574-L593
+    #   date: (string)
+    #   heQualificationType: https://github.com/DFE-Digital/qualified-teachers-api/blob/main/docs/api-specs/v2.json#L840-L930
+    def qualification_params
+      {
+        providerUkprn: nil,
+        countryCode: country_code_for(qualification.institution_country),
+        subject: qualification.title,
+        class: "NotKnown",#https://github.com/DFE-Digital/qualified-teachers-api/blob/main/docs/api-specs/v2.json#L574-L593
+        date: qualification.certificate_date,
+        heQualificationType: "Unknown"  #https://github.com/DFE-Digital/qualified-teachers-api/blob/main/docs/api-specs/v2.json#L840-L930
+      }
+    end
+
+    def country_code_for(_country)
+      # TODO: mapping these. Probably need to make country an autocomplete
+      # register's mapping -> https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/lib/hesa/code_sets/countries.rb
+      "CA" # hard code Canada for now
+    end
```

Comment:
We've got this list in the app already https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/main/public/location-autocomplete-canonical-list.json but it sounds like it might not line up with the ones in DQT, so we may need to maintain a separate one or change our country list to match DQT.